### ISSUE TITLE
Error using @frctl/twig for components engine

### ISSUE THREAD

**Issue by john-henry** [area17/fractal-axe-a11y#1]
I'm using @frctl/twig for my components engine and getting this error

> TwigException: Twig.expression.type.string cannot follow a Twig.expression.type.variable at template:4 near ''/axe-a11y/a11y.js'...'

> Error: TwigException: Twig.expression.type.string cannot follow a Twig.expression.type.variable at template:4 near ''/axe-a11y/a11y.js'...'
>     at /var/www/html/node_modules/@frctl/twig/src/adapter.js:156:24
>     at new Promise ()
>     at TwigAdapter.render (/var/www/html/node_modules/@frctl/twig/src/adapter.js:134:16)
>     at ComponentSource._wrapInLayout (/var/www/html/node_modules/@frctl/fractal/src/api/components/source.js:277:43)
>     at _wrapInLayout.next ()
>     at onFulfilled (/var/www/html/node_modules/co/index.js:65:19)

I presume twig is just not supported?

Twig file is simple enough 

```
<button type="button" class="inline-block px-8 py-3 font-medium text-center rounded-lg shadow hover:shadow-md transform hover:-translate-y-1 transition-all duration-200 transition-ease-out text-white bg-blue hover:bg-light-blue">{{ label }}</button>
```
